src/interfaces.js

```diff
@@ -132,5 +132,5 @@ interface ConfigureRouter {
   /**
   * Implement this hook if you want to add sub-routes to your view.
   */
-  configureRouter(config: RouterConfiguration, router: Router);
+  configureRouter(config: RouterConfiguration, router: Router): any;
```

Comment:
Then why don't You mark it with union type? As I understand union types are supported by Babel 6 (and TypeScript).